lib/LDA2Vec.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
This comment is valid throughout this file. Please follow documentation standards, some params are not documented and I think the method descriptions are too small, they don't describe the functionalities. for example, in the train method, it would be nice if you briefly explain the approach and add in-line comments when the line of code is too vague.